tools/flang2/flang2exe/llutil.cpp

```diff
@@ -2377,9 +2457,13 @@ write_operand(OPERAND *p, const char *punc_string, int flags)
         if (p->ll_type)
           new_op->ll_type = p->ll_type;
       } else {
-        new_op = make_var_op(p->val.sptr);
-        if (p->ll_type)
-          new_op->ll_type = ll_get_pointer_type(p->ll_type);
+	if (STYPEG(p->val.sptr) == ST_PARAM) {
+	  new_op = make_param_op(p->val.sptr);
+	} else {
+	  new_op = make_var_op(p->val.sptr);
+	  if (p->ll_type)
+	    new_op->ll_type = ll_get_pointer_type(p->ll_type);
+	}
```

Comment:
Alignment.

Reply:
Thanks for your comment. Incorporating it.